frontend/src/features/documents/list/DocumentListView.tsx

```diff
@@ -100,10 +115,29 @@ export const DocumentListView: React.FunctionComponent<IDocumentListViewProps> =
     }
   };
 
+  const onUploadSuccess = () => {
+    handleModalUploadClose();
+    props.refreshDocumentList();
+  };
+
   return (
-    <Styled.ListPage>
-      <Styled.Scrollable vertical={true}>
-        <Styled.PageHeader>Documents</Styled.PageHeader>
+    <div>
+      <Section
+        header={
+          <Row>
+            <Col xs="auto">Documents</Col>
+            {hasClaim(Claims.RESEARCH_ADD) && (
```

Comment:
is this the right claim?

Reply:
I dont think so. It suffers from the same claims issues other document sections has